package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/SkeletonPlaceholder.js",
```

Comment:
to run this branch here on my machine I had to change the path to src/SkeletonPlaceholder.tsx